### ISSUE TITLE
version 0.2.2: Not persisting Suites settings anymore via GUI

### ISSUE THREAD

**Issue by r-roos** [lucianopuccio/golem#37]
First of all met me say: Awesome job on this framework! I am really digging it.

I want to report some bugs I encountered moving from v0.2.1 -> v0.2.2 (new GUI layout)
1. The Suites settings wont show any suggestions anymore for drivers like eg. "chrome or chrome-headless"
2. Selected data like driver and test cases are not persisted anymore, pressing save generates the following error: 
```
File "/home/xxxxxx/dev/python/venvs/doodle/lib/python3.5/site-packages/golem/gui/__init__.py", line 626, in save_suite
    environments = request.json['environments']
KeyError: 'environments'
```

**Comment by r-roos** [comment]
I cannot reproduce this problem using the latest source version, it seems isolated to the version hosted on pypi

**Comment by lucianopuccio** [comment]
> Awesome job on this framework! I am really digging it.
Thanks! much appreciated.

I tried to reproduce this on 0.2.2 and couldn't, could be a matter of cache. Anyway I will make a release soon, hope it get's fixed by it

**Comment by r-roos** [comment]
Could indeed be a caching thing when I switched from 0.2.1 to 0.2.2.

As noted earlier, with the latest source version I did not expierience this problem yet. Most likely this issue can be considered closed

**Comment by r-roos** [comment]
i consider the issue closed after using latest source versions and 0.3.0
